app/hotels/src/navigation/allHotels/SearchResultsScreen.js

```diff
@@ -52,8 +52,8 @@ class SearchResultsScreen extends React.Component<Props> {
 
     const text =
```

Comment:
Maybe that should be `translationId` instead of `text`. WDYT?

Reply:
I think I should change it to just pass the `Translation` component, since that is what we normally do

Reply:
Yes, that makes sense.